src/Command/ResetPasswordRemoveExpiredCommand.php

```diff
@@ -20,15 +20,15 @@
  */
 class ResetPasswordRemoveExpiredCommand extends Command
 {
-    protected static $defaultName = 'reset-password:remove-expired';
+    private static $commandName = 'reset-password:remove-expired';
 
     private $cleaner;
 
     public function __construct(ResetPasswordCleaner $cleaner)
     {
         $this->cleaner = $cleaner;
 
-        parent::__construct();
+        parent::__construct(self::$commandName);
```

Comment:
The only place we're using the `reset-password:remove-expired` string is for the command name in the constructor. I think it would be better if we just remove the property all together.
```suggestion
        parent::__construct('reset-password:remove-expired');
```